python/scripts/archive_repositories.py

```diff
@@ -77,19 +77,17 @@ def get_config_for_organization(github_organization_name: str) -> tuple[str, lis
         f"Unsupported Github Organization Name [{github_organization_name}]")
 
 
-def archive_inactive_repositories_by_date_and_type(github_token: str, organization_name: str, archive_date: datetime,
+def archive_inactive_repositories_by_date_and_type(github_service: GithubService, archive_date: datetime,
                                                    allow_list: list[str],
-                                                   repo_type_to_archive: str):
-    moj_gh = MojGithub(org=organization_name, org_token=github_token)
-
+                                                   repository_type: str):
     repos = [
         repo
-        for repo in moj_gh.get_unarchived_repos(repo_type_to_archive)
+        for repo in github_service.get_repositories_to_consider_for_archiving(repository_type)
         if ready_for_archiving(repo, archive_date)
     ]
 
     logging.info(
-        f"Beginning archive of inactive repositories for GitHub organization: {organization_name}")
+        f"Beginning archive of inactive repositories for GitHub organization: {github_service.organisation_name}")
```

Comment:
A repeat of the conversation on Friday 😄 

If we choose the English spelling (which, in my opinion, we should), then we should make changes to all instances.

Reply:
Agreed, we should standardise and be consistent! This PR was doing enough already, though 😂

Are we okay to leave this, and we'll fight that battle another day? ⚔️

I'll leave the conversation resolution up to 👀